### ISSUE TITLE
Handle unauthorized structures

### ISSUE THREAD

**Issue by mcgibbon** [wgilpin/recruitment-ascee#6]
Will need to do something when we can't query structure data for asset locations. Probably have a fallback structure whose name is Unknown Structure in a system called Unknown System in a region called Unknown Region?

**Comment by wgilpin** [comment]
Yeah it's a bit shit of CCP. Given that I can see the name and owner of a structure in space

**Comment by mcgibbon** [comment]
Ah I'll name it "Unknown Structure <number>" so we can differentiate between them.
